README.md

```diff
@@ -35,7 +35,7 @@ for easy to use recipes and integration with sbt, visit [this project](https://g
 ## Getting started
 
 This library is currently available for Scala binary versions 2.12, 2.13 and 3.2 on JVM/JS/Native.  
-This library is architecured in a microkernel fashion and all the main kubernetes stuff are implemented/generated in pure scala, and integration modules are provided separately.  
+This library is designed in a microkernel fashion and all the main kubernetes stuff are implemented/generated in pure scala, and integration modules are provided separately.  
```

Comment:
The most obvious word would be `architectured` but it isn't much of a word ([m-w doesn't believe it exists](https://www.merriam-webster.com/dictionary/architectured)), and even [wiktionary for `architectured`](https://en.wiktionary.org/wiki/architectured#:~:text=architectured%20(comparative%20more%20architectured%2C%20superlative,as%20if%20by%20an%20architect)) just suggests `designed`.

Most people would use `architected` instead of `architectured`